src/components/appComponents/booksComponents/bookListComponents/BookItem.js

```diff
@@ -13,11 +12,11 @@ function BookItem(props) {
 
   function handleClick(e) {
     const payload = Number(e.target.dataset.bookid);
-    dispatch(removeBook(payload));
+    dispatch(removeBookAPI(payload));
   }
 
   return (
-    <div className="row book-item">
+    <div className="book-item">
```

Comment:
- [ ] You can use also this in separate component and add the filter to your reducer to make it clean.